package.json

```diff
@@ -5,6 +5,7 @@
   "type": "module",
   "main": "index.js",
   "scripts": {
+    "start": "node app.js",
```

Comment:
Adds a little script to start the app using `npm run start`

Reply:
Thank you for the changes you made. I am considering adding a different library for the subtitle library soon to allow translations into other languages.
[libary](https://github.com/microlinkhq/youtube-dl-exec)